dev-util/android-sdk-update-manager/android-sdk-update-manager-26.1.1.ebuild

```diff
@@ -0,0 +1,109 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="8"
+PYTHON_COMPAT=(python3_{10..11})
```

Comment:
If it can compile with python3.9 please maintain it. We use to set at least 3 versions of python